reference/7.1/Microsoft.PowerShell.Core/About/about_Functions_Advanced_Parameters.md

```diff
@@ -122,6 +122,128 @@ Param(
 )
 ```
 
+## Switch parameters
+
+Switch parameters are parameters with no parameter value. They're effective
+only when they're used and have only one effect.
+
+For example, the **NoProfile** parameter of **powershell.exe** is a switch
+parameter.
+
+To create a switch parameter in a function, specify the `Switch` type in the
+parameter definition.
+
+For example:
+
+```powershell
+Param([Switch]<ParameterName>)
+```
+
+Or, you can use an another method:
+
+```powershell
+Param(
+    [Parameter(Mandatory=$false)]
+    [Switch]
+    $<ParameterName>
+)
+```
+
+Switch parameters are easy to use and are preferred over Boolean parameters,
+which have a more difficult syntax.
+
+For example, to use a switch parameter, the user types the parameter in the
+command.
+
+`-IncludeAll`
+
+To use a Boolean parameter, the user types the parameter and a Boolean value.
+
+`-IncludeAll:$true`
```

Comment:
I think you were trying to show why you shouldn't use a bool parameter and use a switch instead, so this example should be

```suggestion
`-IncludeAll $true`
```

You should probably have a separate example where you want to explicit set the state of a switch using the `-IncludeAll:$false` syntax.